mmcv/ops/csrc/pytorch/fused_bias_leakyrelu.cpp

```diff
@@ -0,0 +1,25 @@
+// Modified from
+// from https://github.com/rosinality/stylegan2-pytorch/blob/master/op/fused_bias_act.cpp
+#include "pytorch_cpp_helper.hpp"
+
+#ifdef MMCV_WITH_CUDA
+torch::Tensor fused_bias_leakyrelu_op(const torch::Tensor& input, const torch::Tensor& bias, const torch::Tensor& refer,
+    int act, int grad, float alpha, float scale);
+
+#define CHECK_CUDA(x) TORCH_CHECK(x.type().is_cuda(), #x " must be a CUDA tensor")
```

Comment:
Some macros have been defined in [pytorch_cpp_helper.hpp](https://github.com/open-mmlab/mmcv/blob/005c408748446bab1615873c46ae65a3a7124a29/mmcv/ops/csrc/pytorch_cpp_helper.hpp).
We can directly use them